t/webservice.t

```diff
@@ -212,6 +210,42 @@ transactionally {
 
 }
 
+transactionally {
+
+    subtest "rename-dataset exception" => {
+        my $model-version = 'SingleSHL';
+        ok my $datasets = $ws.get-datasets($user, $model-version), "Get $model-version datasets";
+        for @$datasets -> $dataset {
+            note $dataset[0];
+        }
+
+        throws-like { $ws.rename-dataset($user, 'TestSingle', 'Agrammon6Testing') },
+            X::Agrammon::DB::Dataset::AlreadyExists,
+            "Rename dataset fails for existing new dataset name";
+    }
+
+}
+
+transactionally {
+
+    subtest "rename-dataset exception" => {
+        throws-like { $ws.rename-dataset($user, 'TestSingle', 'TestSingle') },
+            X::Agrammon::DB::Dataset::RenameFailed,
+            "Rename dataset fails if old and new name are identical";
+    }
```

Comment:
A question of taste, but `throws-like` already does a `subtest` call for the things it asserts, so we can have slightly simpler output without the outer layer of `subtest` here.

Reply:
Removed